docs/es-overview.asciidoc

```diff
@@ -2,150 +2,122 @@
+Elastic Security combines SIEM threat detection features with endpoint
+prevention and response capabilities in one solution. These analytical and
+protection capabilities, leveraged by the speed and extensibility of
+Elasticsearch, enable analysts to defend their organization from threats before
+damage and loss occur.
 
 Elastic Security provides the following security benefits and capabilities:
 
-* A powerful detection engine to identify attacks and system misconfiguration
-* Interactive visualizations to investigate attacks
-* Accelerated response with embedded case management and automated actions
-* Detection of signatureless attacks with prebuilt anomaly detection jobs and detection rules
+* A detection engine to identify attacks and system misconfigurations
+* A workspace for event triage and investigations
+* Interactive visualizations to investigate process relationships
+* Inbuilt case management with automated actions
+* Detection of signatureless attacks with prebuilt machine learning anomaly jobs
+and detection rules
+
+[discrete]
+== Elastic Security components and workflow
 
 The following diagram provides a comprehensive illustration of the Elastic Security workflow.
 
 [role="screenshot"]
-.Elastic Security workflow
 image::images/workflow.png[Elastic Security workflow]
 
-[discrete]
-== Elastic Security components
-
-The {security-app} in {kib} provides an interactive workspace for security teams to
-triage events and perform initial investigations. Additionally,
-<<machine-learning, {ml} {anomaly-jobs}>> and
-<<detection-engine-overview, detection engine rules>> provide ways to
-automatically detect suspicious activity across your entire fleet of servers
-and workstations.
-
-Elastic Security requires the following {stack} components:
+Here's an overview of the flow and its components:
+
+* Data is shipped from your hosts to {es} via beat modules and the Elastic
+Agent:
+** <<install-endpoint, {es-sec-endpoint}>> - Elastic agent integration that
+protects your hosts against malware and ships these data sets:
+***  *Windows*: process, network, file, DNS, registry, DLL and driver loads,
+malware security detections 
+*** *Linux/macOS*: process, network, file
+** https://www.elastic.co/integrations?solution=security[Beat modules]: {beats}
+are lightweight data shippers. Beat modules provide a way of collecting and
+parsing specific data sets from common sources, such as cloud and OS events,
+logs, and metrics. Common security-related modules are listed
+<<enable-beat-modules, here>>. 
+* The Security app in {kib} is used to manage the *Detection engine*,
+*Cases*, and *Timeline*, as well as administer hosts running Endpoint Security:
+** Detection engine - Automatically searches for suspicious host and network
+activity:
```

Comment:
I think we can add something at the end that leads up to the following bullets (e.g., `"...Automatically searches for suspicious host and network activity via the following tools"` or something to that effect.

Reply:
changed to: `Detection engine - Automatically searches for suspicious host and network
activity via:`